lib/command.rb

```diff
@@ -12,6 +14,24 @@ def self.synopsis
 
       def execute
         with_target_vms(nil, {:single_target=>true}) do |machine|
+
+	  options = {}
+          opts = OptionParser.new do |o|
+	    o.banner = "Usage: vagrant adbinfo [options]"
+            o.separator ""
+
+            o.on('box-version', 'box-version', 'Prints the version of Virtual Machine') do |boxversion|
+	      options[:boxversion] = boxversion;
+	    end
+          end
+          
+          # parse the options
+          argv = parse_options(opts)
```

Comment:
Missing `parse_options` method. Couldn't find it anywhere. 
Also no need of comment as the line is self-explanatory.